multicluster/test/e2e/README.md

```diff
@@ -0,0 +1,14 @@
+# Running the Antrea Multicluster end-to-end tests
+
+## Creating the test kubernetes cluster set
+
+The tests must be run on an actual Kubernetes cluster set, and there must be at least three clusters. We can create the clusters by [kubeadm](https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm) or the [instructions](https://github.com/antrea-io/antrea/blob/main/test/e2e/README.md) or other ways.
+
+The three clusters is called leader cluster, west cluster and east cluster. And the CIDR of pod in each clusters should not same. For example, the cidr of pod could be `192.168.0.1/22`,`192.168.4.1/22` and `192.168.8.1/22` in three clusters.
```

Comment:
is called -> are called

Reply:
CIDR of pod in -> Pod CIDR of

should not be the same

cidr of pod -> Pod CIDRs